src/Xive/Comb/FileComb.cs

```diff
@@ -62,7 +64,40 @@ public FileComb(string root, string name, Func<ICell, ICell> cellWrap, Func<IXoc
 
         public ICell Cell(string name)
         {
-            return this.comb.Value().Cell(name);
+            ICell result;
+            if (name.Equals("_guts.xml"))
+            {
+                var patch = new Directives().Add("items");
+
+                foreach (var file in Directory.GetFiles(
+                        Path.GetDirectoryName(this.comb.Value().Name()),
+                        "*.*",
+                        SearchOption.AllDirectories))
+                {
+                    patch.Add("item")
+                        .Add("name")
+                        .Set(Path.GetFileName( file)) //file.Substring((this.name + "/").Length))
```

Comment:
Remove comment (and unneeded space in brackets)

Reply:
comment removed - which unneeded space?

Reply:
the leading space in front of `file`

Reply:
thanks, sorry was focussed on the foreach brackets